test/promql/tests/data_plane/pipeline_to_snapshot_test.yaml

```
@@ -31,11 +31,42 @@ tests:
             summary: PipelineRunFinish to SnapshotInProgress time exceeded
             description: >
               Time from pipeline run finished to snapshot marked in progress has been over
-              30s for more than 10% of requests during the last 5 minutes on cluster
+              30s for more than 10% of requests during the last 10 minutes on cluster
               cluster01
             alert_routing_key: integration-service
             runbook_url: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/integration-service/pipeline_to_snapshot_exceeded.md
 
+  # Scenario where one cluster crosses the 10% threshold after 7 minutes and another does not
+- interval: 1m
+  input_series:
+    # Simulating data from Cluster 1
+    - series: 'integration_svc_response_seconds_bucket{le="30", namespace="integration-service",  source_cluster="cluster01"}'
+      values: '7+10x10'  # 10 requests took less than 30s
```

```suggestion
      values: '7+10x20'
```

We now evaluate at 21 minutes, but we have time series for only 11 minutes. We need to extend the time series.  I see now that other test cases were not updated with longer time series either.